.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "eslint:recommended",
+  "env": {
+    "es6": true,
+    "node": true,
+    "mocha": true
+  },
+  "rules": {
+    "space-before-function-paren": ["error", "never"],
+    "standard/no-callback-literal": "off",
+    "arrow-spacing": "error",
+    "arrow-parens": ["error", "as-needed"],
```

Comment:
This [rule](https://eslint.org/docs/rules/arrow-parens#as-needed) should probably be changed as it conflicts with the [codacy linting](https://app.codacy.com/manual/cloudevents/sdk-javascript/pullRequest?prid=5358785) rules.

Reply:
Could you change it to follow codacy rules?